pkgs/tools/audio/tts/default.nix

```diff
@@ -12,19 +12,18 @@
 # For now, for deployment check the systemd unit in the pull request:
 #   https://github.com/NixOS/nixpkgs/pull/103851#issue-521121136
 #
+# Check the latest release for compatible models:
+#   https://github.com/coqui-ai/TTS/releases/tag/v0.0.11
 
 python3Packages.buildPythonApplication rec {
   pname = "tts";
-  # until https://github.com/mozilla/TTS/issues/424 is resolved
-  # we treat released models as released versions:
-  # https://github.com/mozilla/TTS/wiki/Released-Models
-  version = "0.0.9";
+  version = "0.0.11";
 
   src = fetchFromGitHub {
-    owner = "mozilla";
+    owner = "coqui-ai";
     repo = "TTS";
     rev = "df5899daf4ba4ec89544edf94f9c2e105c544461";
```

Comment:
shouldn't the rev also change?

Reply:
Meh, `nix-update` isn't updating the hash and I don't know why.

Reply:
```suggestion
    rev = "v{version}";
```

Reply:
> Eh, the rev has no version reference....